### PR TITLE
basic cement cli application with support for playing a file with vlc

### PR DESCRIPTION
**Pull request by jakebuller**

### CODE REVIEW

File: roger.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# ideally python will have a nicer system for dependencies
+# this will do in the meantime
+try:
+    from cement.core.foundation import CementApp
+except ImportError:
+  exit("ERROR! Could not import 'cement'. Try: sudo pip install cement")
+
+# this could be moved into a player library or something to that effect
+vlc = "/Applications/VLC.app/Contents/MacOS/VLC"
```
Comment (travisMiehm) [inline roger.py]: Can you generalize this to use your PATH environment variable?

File: roger.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import subprocess
+
+# ideally python will have a nicer system for dependencies
+# this will do in the meantime
+try:
+    from cement.core.foundation import CementApp
+except ImportError:
+  exit("ERROR! Could not import 'cement'. Try: sudo pip install cement")
```
Comment (travisMiehm) [inline roger.py]: Is this for when the package isn't present?  So ugly.

File: roger.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+app.run()
+
+# add application logic
+if app.pargs.file:
+    app.log.info("Received the 'file' option with value '%s'." % app.pargs.file)
+    file = app.pargs.file
+
+    # should refactor this into a nicer class/interface/pythonish pattern
+    subprocess.call([vlc, file])
```
Comment (travisMiehm) [inline roger.py]: How does this behave when the file does not exist?

File: roger.py (hunk 2 above)
Comment (jakebuller) [inline roger.py]: haha yea, hence the comment about dependency management.

File: roger.py (hunk 3 above)
Comment (jakebuller) [inline roger.py]: just fails miserably

File: roger.py (hunk 1 above)
Comment (jakebuller) [inline roger.py]: yea probably, we should figure out a nice way to set up application integrations and environments that will abstract this out

File: roger.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+else:
+    app.log.error("Did not receive a value for 'file' option.")
```
Comment (travisMiehm) [inline roger.py]: I generally like to have a showUsage() function to output the available CLI options and such

**Comment by travisMiehm** [comment]
:+1:

**Pull request merged**